Subject: Fix keyboard scroll scancodes that were clashing with right "Win" key
Date: Fri Mar 18 10:45:24 CET 2005
From: Vojtech Pavlik <vojtech@suse.cz>
References: OSDL: #4361
===================================================================


ChangeSet@1.2091, 2005-03-18 10:43:28+01:00, vojtech@suse.cz
  input: Fix fast scrolling scancodes in atkbd.c
  
  Signed-off-by: Vojtech Pavlik <vojtech@suse.cz>


 atkbd.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)


diff -Nru a/drivers/input/keyboard/atkbd.c b/drivers/input/keyboard/atkbd.c
--- a/drivers/input/keyboard/atkbd.c	2005-03-18 10:43:46 +01:00
+++ b/drivers/input/keyboard/atkbd.c	2005-03-18 10:43:46 +01:00
@@ -171,9 +171,9 @@
 	unsigned char set2;
 } atkbd_scroll_keys[] = {
 	{ ATKBD_SCR_1,     0xc5 },
-	{ ATKBD_SCR_2,     0xa9 },
-	{ ATKBD_SCR_4,     0xb6 },
-	{ ATKBD_SCR_8,     0xa7 },
+	{ ATKBD_SCR_2,     0x9d },
+	{ ATKBD_SCR_4,     0xa4 },
+	{ ATKBD_SCR_8,     0x9b },
 	{ ATKBD_SCR_CLICK, 0xe0 },
 	{ ATKBD_SCR_LEFT,  0xcb },
 	{ ATKBD_SCR_RIGHT, 0xd2 },
